setup.py

```diff
@@ -66,28 +27,29 @@ def call(arguments):
 
 
 def get_config(scorep_config):
```

Comment:
that looks like you did `scorep-init-preload` again. can you check if this will help you here?

Reply:
Partly I think. For the MPI runtime, I am using preloading, but not for the non mpi module. In this case `scorep_init.c` is directly linked into the module.

The Idea is here to generate the init once with some Score-P features. So the `scorep_init.c` does not needed to be build for every experiment.

I am not sure how `scorep-preload-init` can be used here. It might be an Idea to allow the user to manually specify their `LD_PRELOAD`, which might then come from `scorep-preload-init`.

Another way would be to always generate the `LD_PRELOAD` with `scorep-preload-init`. But this would require regularly builds, and maybe clean ups. I am not really happy with this Idea.

What do you think?

Best,

Andreas